### PR TITLE
Replace layer5 logo with service mesh patterns logo in workflow

### LINKED ISSUE #102: Replace the Logo of Layer5 with Service-Mesh-Patterns in config.yml workflow
### Description

Currently, we are using the Logo of Layer5 instead of service-mesh-patterns. This repository is under the organization service-mesh-patterns, so, there should be the image of service-mesh-patterns.

**How to resolve this?**
- Go to .github and replace this welcome image with the new one: 

Service-Mesh-Patterns-celebration

- Download this image, and upload with the correct name for future reference. The name of the image should be Service-Mesh-Patterns-celebration
- After uploading the image, and removing the previous image, go to .github/config.yml folder and update the reference to previous image to the new image in the following part:

```
firstPRMergeComment: >
  Thanks for your contribution to the Layer5 and Meshery community! :tada:
   
  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; :star: Please leave a [star on the project](../stargazers). :smile:
```
- That's it.

-----
#### Contributor Guides and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing](https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/CONTRIBUTING.md)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by senali-d**
**Description**

This PR fixes #102

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 93a9cb2027fcb43cccd7f0525223205508a326be at: https://6444ede3954f5a012f17a9df--service-mesh-patterns.netlify.app

**Comment by thisiskaransgit** [comment]
@senali-d, Better use these, for both color schemes
Untitled (4)
Untitled (3)

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@senali-d, Better use these, for both color schemes
Untitled (4)
Untitled (3)

**Comment by HetviSoni** [comment]
I would like to work on this issue.

**Comment by senali-d** [comment]
@thisiskaransgit Requested change was done.

**Comment by thisiskaransgit** [comment]
Hey @senali-d, the svgs were not passing the  quality check, @ritiksaxena124 has provided the recommended one, kindly update the images once more, and we are good to go.

Patterns-dark

Patterns-light

File: .github/config.yml [hunk 1]
```diff
@@ -15,14 +15,17 @@ newPRWelcomeComment: >
...
 firstPRMergeComment: >
-   &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Thank you for contributing to the Layer5 community! :tada:
-  \
-  \
-  ![Congrats!](https://github.com/layer5io/layer5/blob/master/.github/welcome/Layer5-celebration.png)
-  \
-  \
-  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; :star: Please leave a [star on the project](../stargazers). :smile:
+  Thank you for contributing to the Layer5 community! :tada:
+
```
Comment (HetviSoni) [inline .github/config.yml:19]: As in meshery.io repo we have "Thanks for your contribution to the Layer5 **and Meshery**  community!"
so here we can have same here or maybe only Meshery
For reference: https://github.com/meshery/meshery.io/blob/master/.github/config.yml

**Comment by l5io** [comment]
🚀 Preview for commit 12080b1e22c1231147579970ea4d796d91f4c0a2 at: https://646787949ea3ca22cd39aa02--service-mesh-patterns.netlify.app

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**